board/app/models/thread.php

```diff
@@ -1,122 +1,180 @@
 <?php
 
-class Thread extends AppModel{
-
+class Thread extends AppModel 
+{
 /* 
-*The following constants are declared to avoid magic numbers.
-*Avoid using magic numbers so others couuld
-*   understand what that number is all about.
+* Avoid using magic numbers so others could understand what that number is all about.
 */
-    const MIN_TITLE_LENGTH = 1;
-    const MAX_TITLE_LENGTH = 30;
+const MIN_TITLE_LENGTH = 1;
+const MAX_TITLE_LENGTH = 30;
 
-//Thread Length Validation
+    /* 
+    * Thread Length Validation
+    */
     public $validation = array(
-        'title' => array(
-            'length' => array(
-                'validate_between', self::MIN_TITLE_LENGTH, self::MAX_TITLE_LENGTH
+    'title' => array(
+        'length' => array(
+            'validate_between', self::MIN_TITLE_LENGTH, self::MAX_TITLE_LENGTH
+            ),
+        ),
+
+    /* 
+    * Category Validation
+    */
+    'category' => array(
+        'length' => array(
+            'validate_between', self::MIN_TITLE_LENGTH, self::MAX_TITLE_LENGTH
             ),
         ),
     );
 
-    public function create(Comment $comment) 
+    public function create(Comment $comment)
```

Comment:
Why do you have a create comment on the model of thread? I think this should be in the comment model po :bow: